File: strings/prob344.py
class Solution(object):
    def reverseString(self, s):
        """
        :type s: List[str]
        :rtype: None Do not return anything, modify s in-place instead.
        """
        m = len(s)
        if m <= 1:
            return
        for i in range(m // 2):
            s[i], s[m - 1 - i] = s[m - 1 - i], s[i] # swap
        return

File: strings/prob344.kt
class Solution {
    fun reverseString(s: CharArray): Unit {
        
        val n = s.size
        var l = 0
        var r = n-1
        while (l < r) {
            var tmp = s[l]
            s[l] = s[r]
            s[r] = tmp
            l++
            r--
        } 
        
        
    }
}
